docker/sqlg/gremlin-server.yaml

```diff
@@ -1,20 +1,18 @@
 host: localhost
 port: 8182
+channelizer: org.apache.tinkerpop.gremlin.server.channel.WsAndHttpChannelizer
 graphs: { graph: /secrets/sqlg.properties }
 scriptEngines:
   {
     gremlin-groovy:
       {
         plugins:
           {
-            ? org.apache.tinkerpop.gremlin.server.jsr223.GremlinServerGremlinPlugin
-            : {},
+            org.apache.tinkerpop.gremlin.server.jsr223.GremlinServerGremlinPlugin:
```

Comment:
I find a bit confusing mixing YAML flow and block syntax, but it's a personal preference, ok with me if we leave it as it is.

Reply:
Yes, totally agree that it is a bit odd, however it is actually how most of Tinkerpop's Gremlin Server examples [are written](https://github.com/apache/tinkerpop/blob/master/gremlin-server/conf/gremlin-server.yaml). I guess I would be in favor of leaving as-is so that it more closely resembles the Tinkerpop docs / tutorials.

Reply:
makes sense, I also prefer to stay close to the official docs